src/github_utils.py

```diff
@@ -0,0 +1,132 @@
+import json
+import os
+import requests
+from github import Github
+
+def get_pull_request_diff():
+    """Fetch the pull request diff content from GitHub"""
+    g = Github(os.getenv('GIT_TOKEN'))
```

Comment:
Consider adding error handling for network requests, especially for `requests.get(diff_url)`. Use try-except blocks to gracefully handle potential exceptions like `requests.exceptions.RequestException`.

```suggestion
|
``````python
try:
response = requests.get(diff_url)
response.raise_for_status()  # Raise HTTPError for bad responses (4xx or 5xx)
return response.text
except requests.exceptions.RequestException as e:
print(f"Error fetching diff content: {e}")
return None # Or handle the error as appropriate
```